### PR TITLE
Add bart-base

### LINKED ISSUE #5004: [🚀 Feature request] upload bart-base checkpoint
# 🚀 Feature request

`bart-base`  checkpoint is now available in fairseq repo 
https://github.com/pytorch/fairseq/tree/master/examples/bart#pre-trained-models
so now it can be made available here  `facebook/bart-base` 

@sshleifer

### PR DESCRIPTION
**Pull request by sshleifer**
Also adds two `@slow` integration test for fill mask tasks.
I made some guesses on the correct conversion, which will hopefully be confirmed by the authors in this [issue](https://github.com/pytorch/fairseq/issues/2242)

### CODE REVIEW

**Review by julien-c — APPROVED** [review]

**Comment by julien-c** [comment]
add a model card too if needed/useful

**Comment by sshleifer** [comment]
Will and model cards for this and bart-large at some point.
